### DISCUSSION TITLE
getting error while trying t run ui test app

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#2842] [Q&A] [answered]
getting error while trying t run ui test app  https://github.com/iTwin/itwinjs-core/tree/master/test-apps/ui-test-app#client-configuration

![image](https://user-images.githubusercontent.com/53996840/144823452-a058e561-3f24-4266-87bb-664263de62f9.png)

![image](https://user-images.githubusercontent.com/53996840/144825409-c90659b2-4b96-494d-9b88-e6c02ed9836c.png)

**Comment by calebmshafer** [comment, marked as answer]
@Rahul-0108 the ui-test-app uses the new [projects-clients](https://github.com/iTwin/projects-client) which uses the iTwin Platform Projects API. That API requires the `projects:read` scope to be specified.
